### DISCUSSION TITLE
Is it possible to use any MsBuild parameter to override solution name in parameters.json file?

### DISCUSSION THREAD

**Discussion by voroninp** [nuke-build/nuke#1060] [Q&A] [answered]
I see that `StronglyTypedSolutionGenerator` tries to resolve solution using values from `parameters.json` file.
But if I change directory structure while copying src to docker container, build project cannot locate solution anymore because its relative path has changed.

Is there a magic MsBuild property to set which will [be used by generator](https://platform.uno/blog/using-msbuild-items-and-properties-in-c-9-source-generators/) so it does not resolve the path from `parameters.json`?

**Comment by matkoch** [comment, marked as answer]
There's no such way.
